### PR TITLE
Update SQL Version Call

### PR DESCRIPTION
**Pull request by djekl**
This is causing an issue for Postgres, when you select `version()`, its returning the result as `version`. Setting the `as` on the statement fixes this issue.

### CODE REVIEW

**Comment by syntafin** [comment]
Almost 4 months without reaction of maintainer :(

**Review by syntafin — APPROVED** [review]

**Comment by djekl** [comment]
@syntafin You can simply use my fork of the code until this is approved and merged.

Add my fork of the code as a repo
```composer config repositories.Kussie/nova-version-card '{"type": "vcs", "url": "https://github.com/djekl/nova-version-card"}' --file composer.json```

Then require my version, as a minor version bump
```composer require "kussie/nova-version-card"@"dev-patch-1 as v0.1.2"```

**Comment by syntafin** [comment]
@djekl I checked your fix and applied it already, would just be easier to use it without addition config.

**Comment by djekl** [comment]
@syntafin 100% agree. Just need @Kussie to merge it, and tag a new release

**Comment by Kussie** [comment]
Apologies for the delay.  This has now been merged

**Comment by djekl** [comment]
@Kussie Thanks for merging. You just need to tag a new release, and I can then remove my fork

**Comment by Kussie** [comment]
@djekl Yep v0.1.1 was just tagged and published

**Pull request merged**
